backend/src/main/java/kernel360/techpick/core/model/folder/FolderStructure.java

```diff
@@ -0,0 +1,48 @@
+package kernel360.techpick.core.model.folder;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import kernel360.techpick.core.model.common.TimeTracking;
+import kernel360.techpick.core.model.user.User;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Table(name = "folder_structure")
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class FolderStructure extends TimeTracking {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@Column(columnDefinition = "longblob", nullable = false)
+	private String structure;
```

Comment:
p3: 지금은 구조를 동적으로 사용하지 않아 전혀 문제 없지만 
혹시 서버에서 저장된 structure String을 비직렬화 해야 하는 경우, 
이렇게 하면 JSON 파싱을 자동으로 할 수 있으니 참고용으로 공유함. 
```suggestion
    @Column(name = "Structure", nullable = false)
    @JdbcTypeCode(SqlTypes.JSON)
    private Structure structure;
```